pom.xml

```diff
@@ -48,12 +48,6 @@
       <scope>test</scope>
     </dependency>
 
-    <dependency>
```

Comment:
The empty separation line should be where the first test dependency starts, not between two test dependencies

Reply:
Please add an empty line to visually separate the test dependencies from the rest above

Reply:
Just did.